src/utils/check-old-dependencies.ts

```diff
@@ -0,0 +1,23 @@
+/*!
+ * Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved.
+ */
+
+import { recordWarning } from './warnings';
+
+const oldDependencies = [
+  '@sap/cloud-sdk-util',
+  '@sap/cloud-sdk-analytics',
+  '@sap/cloud-sdk-core',
+  '@sap/cloud-sdk-generator',
+  '@sap/cloud-sdk-test-util'
+];
+
+export function checkOldDependencies(dependenciesInPackageJson: any) {
+  oldDependencies.forEach(oldDependency => checkOldDependency(dependenciesInPackageJson, oldDependency));
```

Comment:
I would assume that the packages themselves have "Deprecated, please use ... instead" as output by using `npm deprecate @sap/cloud-sdk-core`. Not sure if the npm.sap.com registry supports this but this would be the correct place IMO.

Reply:
The DMZ colleague suggest that we wait until the migration of the npm.sap.com, which should happen soon.